upgrade/cnt-2.2.x-to-2.3.0/index.php

```
@@ -314,35 +334,39 @@ public function apply_block()
-        $xoopsDB->exec($sql);
+        if (!$this->execOrFail($sql)) {
+            return false;
+        }
 
-        $sql    = '   SELECT MAX(instanceid) FROM ' . $xoopsDB->prefix('block_instance');
-        $result = $xoopsDB->query($sql);
-        if (!$xoopsDB->isResultSet($result)) {
+        $sql    = '   SELECT MAX(instanceid) FROM ' . $this->db->prefix('block_instance');
+        $result = $this->db->query($sql);
+        if (!$this->db->isResultSet($result) || !($result instanceof \mysqli_result)) {
             throw new \RuntimeException(
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '344', column '23'). (undefined)

(MissingImport)